src/models/item.js

```diff
@@ -0,0 +1,41 @@
+
+const item = class Item {
+    //create a constructor method for Item class
+        constructor(name,quantity,price){
+            //initialize all required variable
+            let _name = name;
```

Comment:
why do you have to prefix your variable names with an underscore symbol?

Reply:
I prefixed the class's variable names to identify them and prevent conflict with variables declared in the class methods